CHANGES/6332.doc

```diff
@@ -1 +1 @@
-Added documentation for the RPM copy API.
+Added documentation for the RPM copy API.
```

Comment:
Are we changing this and 6861 for format-checking reasons?

Reply:
The linter doesn't care and I'm not sure if towncrier does, but yeah I'm just fixing it for consisency because none of the others have this.